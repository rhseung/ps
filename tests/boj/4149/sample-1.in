18991325453139
