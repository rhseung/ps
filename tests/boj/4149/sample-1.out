3
3
13
179
271
1381
2423
